designs/administrator/4.8/getting-started-card/index.md

```diff
@@ -0,0 +1,67 @@
+---
+parent: Administrator
+version: 4.8
+---
+
+# Getting started card
+
+## Background
+A quick starts card exists today on the following landing pages for the Admin and Developer perspectives with the goal to help users learn and use resources in the Console.
+
+- Admin Cluster Overview dashboard
+
+![cluster-overview](img/quick-starts-cluster-overview.png)
+
+- Developer Add page 
+
+![add-page](img/quick-starts-add.png)
+
+## Goal
+As our getting started efforts expand, we realize that we can capture more than just quick starts to help users get started. The goal of this larger Getting Started card is to supply users with many different pathways to learn and use resources in the Console. This card will replace the existing quick start card on both the Add page and Cluster Overview.
+
+## Card design and interactions
+The design of the card will remain consistent on the Admin and Developer perspectives, but the content within each column will vary. The reason for this is because there is featured content we would like users to see depending on the perspective they land on.
+
+![card](img/card-design.png)
```

Comment:
I guess I'd mention that because this image calls out these are on 'connected clusters' I sort of expected an equivalent image or guidance on what to do if the cluster is disconnected, which doesn't seem to be uncommon. Can there be less than 3 columns?

Reply:
@itsptk We decided the same content would be shown if you have a connected or disconnected cluster, so I might just cut out the pink title there.

Yeah there can be less than 3 columns depending on content you are dealing with, if validation is included (which then a column would disappear) - I should probably add that in there